docs/overview/build.md

```diff
@@ -23,11 +24,17 @@ The input of the **build** step is a single commit in a git repo and the output
 
 The following is a close approximation of the code we use to take your code repo and turn it into an artifact that can be run inside of AWS Lambda with the Cyclic lambda runtime.
 
-- Install your code's dependencies `npm install`
-- Zip of your code and installed dependencies (for example `node_modules`) recursively
+- Install your code's dependencies (including `devDependencies`) `npm install`
+- Build your app if you have a `build` script defined in `package.json` aka `npm run build`
+- Prune any `devDependencies` to minimize the size of the output zip
+- Zip all of your code and installed dependencies (for example `node_modules`) recursively
 - Stage the zip into AWS in preparation of the **deploy** step
 
-## Lifecycle scripts
+## Common problems
```

Comment:
Should be like a limitations sections instead of problems

Reply:
👍